api/src/domain/types.ts

```diff
@@ -46,6 +47,7 @@ export interface EncryptionDecryptionClient {
   decryptValue: (valueToBeDecrypted: string) => Promise<string>;
 }
 
+export type EncryptTaxId = (UserData: UserData) => Promise<UserData>;
```

Comment:
variable name should be lowercase u

Reply:
It's a type right, should it still be lowercase? I was using the same notation as the `AddNewsLetter` and `AddToUserTesting` below it.